stdlib/src/builtin/file.mojo

```diff
@@ -414,6 +414,14 @@ struct FileHandle:
         """
         self._write(data.unsafe_ptr(), len(data))
 
+    fn write(self, data: Span[UInt8, _, _]) raises:
+        """Write a borrowed sequence of data to the file.
+
+        Args:
+          data: The data to write to the file.
+        """
+        self._write(data.unsafe_ptr().bitcast[Int8](), len(data))
```

Comment:
Is this bitcast necessary? We are currently transitionning to making bytes UInt8 values.

Also since this is the same code as `fn write(self, data: String) raises:` without the bitcast, would it be possible to write generic code that would work for both using traits?